Ошибка: HTTPConnectionPool(host='o', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000161C4FFFC10>: Failed to resolve 'o' ([Errno 11001] getaddrinfo failed)"))
URL: http://o